uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-report.scss

```diff
@@ -35,7 +35,7 @@
 
 
 .chr_leave-report__heading {
-  background-color: $crm-table-heading-bg;
```

Comment:
I am not sure how it was possible. This variable `$crm-table-heading-bg` does not exists at all in Shoreditch. How it was compiled before at all?

![image](https://user-images.githubusercontent.com/3973243/33774622-c30ae120-dc33-11e7-9050-98442ca828d5.png)

Reply:
Is this also because of some changes in other repos and a lack of sync with Shoreditch?

Reply:
Well, my guess is that those who were working in leave and absence extension, they did not have latest org.civicrm.shoreditch repo. Because i had the shoreditch up-to-date and when i tried to run gulp in l&A repo i got the scss compilation error.